legacy/expressions/migrate_test.go

```diff
@@ -154,12 +154,17 @@ func TestMigrateTemplate(t *testing.T) {
 		{old: `@(5 - 4)`, new: `@(5 - 4)`},
 		{old: `@(ABS(5) + MOD(7, 2))`, new: `@(abs(5) + mod(7, 2))`},
 
-		// date+number addition should get converted to datetime_add
+		// datetime+number addition should get converted to datetime_add
 		{old: `@(date.now + 5)`, new: `@(datetime_add(now(), 5, "D"))`},
 		{old: `@(now() + 5)`, new: `@(datetime_add(now(), 5, "D"))`},
 		{old: `@(date + 5)`, new: `@(datetime_add(now(), 5, "D"))`},
 		{old: `@(date.now + 5 + contact.age)`, new: `@(legacy_add(datetime_add(now(), 5, "D"), contact.fields.age))`},
 
+		// date+number addition should get converted to format_date(datetime_add(...))
+		{old: `@(date.today + 5)`, new: `@(format_date(datetime_add(format_date(today()), 5, "D")))`},
+		{old: `@(date.yesterday - 5)`, new: `@(format_date(datetime_add(format_date(datetime_add(now(), -1, "D")), -5, "D")))`},
+		{old: `@(date.tomorrow - 3 + 10)`, new: `@(format_date(datetime_add(format_date(datetime_add(format_date(datetime_add(now(), 1, "D")), -3, "D")), 10, "D")))`},
```

Comment:
us: you're gonna love the new expression syntax!
them: wtf happened to my expressions ????

Reply:
Ugh.. ya, well I guess its not permanent yet?

Reply:
yeah and this is a pretty extreme example